pom.xml

```diff
@@ -19,7 +19,7 @@
         <quarkus.platform.artifact-id>quarkus-bom</quarkus.platform.artifact-id>
         <quarkus.platform.group-id>io.quarkus</quarkus.platform.group-id>
         <quarkus.platform.version>999-SNAPSHOT</quarkus.platform.version>
-        <quarkus.qe.framework.version>1.3.0.Beta11</quarkus.qe.framework.version>
+        <quarkus.qe.framework.version>1.3.0.Beta12</quarkus.qe.framework.version>
```

Comment:
please at least bump it in a separate commit

Reply:
That is my preference as commit message `Cover separate management interface deployed by openshift-extension` doesn't tell me that Framework version has changed. I'm not reviewer here, so feel free to ignore my preference.